src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaSendText = lua_tostring( L, 1 );
+    }
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    QString text(luaSendText.c_str());
```

Comment:
As I've suggested a modification to capture the (unicode) QString directly this line can go and the following would become:

    lua_pushboolean( L, pHost->getKeyUnit()->killKey( itemName ) );

Hope these suggestions are of use...